gitfame/_gitfame.py

```diff
@@ -265,7 +295,16 @@ def _get_auth_stats(
     old = auth_stats
     auth_stats = {}
     for auth, stats in getattr(old, 'iteritems', old.items)():
-      auth_stats[auth2em[auth]] = stats
+      # Some users will change their name over time while keeping the same email address,
+      # such as capitalization or middle names shown.
+      # Handle this by merging together all stats for same email
+      i = auth_stats.setdefault(auth2em[auth], { "loc": 0, "files": set([]), "commits": 0, "ctimes": [] })
+      i["loc"] += stats["loc"]
+      i["files"].update(stats["files"])
+      i["commits"] += stats["commits"]
+      i["ctimes"] += stats["ctimes"]
+      # mutating the returned object sidesteps the need to perform another dict lookup
+      # auth_stats[auth2em[auth]] = i
```

Comment:
haven't reviewed in-depth but looks like this comment means there's still something to address?

Reply:
Ah! Nope - I was attempting to annotate to future readers why there was not a write operation to stash the object back in the dict(). It's not needed because we're operating on the object by reference rather than by value.

Reply:
oh right I understand.

Yeah not really a fan of explaining what seems like an obvious feature of the language.

Will merge soon (and also open another issue to address things like PR build jobs); thanks